apis/metal3.io/v1alpha1/baremetalhost_validation_test.go

```diff
@@ -42,6 +42,97 @@ func TestValidateCreate(t *testing.T) {
 		Name:      "07564256-96ae-4315-ab03-8d34ece60fbb",
 		Namespace: "test-namespace",
 	}
+
+	om_sa_valid := metav1.ObjectMeta{
```

Comment:
Please convert everything to camelCase, Go does not use snake_case.

Reply:
Moved the content inline, so removed these variables.